### PR TITLE
Adding dry-run & fieldmanagers to custom specs

### PR DESCRIPTION
**Pull request by yue9944882**
/cc @brendandburns 

syncing up w/ the latest kubernetes built-in openapi specs

### CODE REVIEW

File: openapi/openapi-generator/Dockerfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ ARG OPENAPI_GENERATOR_USER_ORG=OpenAPITools
 RUN apt-get update && apt-get -y install git python-pip && pip install urllib3==1.24.2
 
 # Install Autorest
-RUN apt-get update && apt-get -qq -y install libunwind8 libicu57 libssl1.0 liblttng-ust0 libcurl3 libuuid1 libkrb5-3 zlib1g
+RUN apt-get update && apt-get -qq -y install libunwind8 libicu57 libssl1.0 liblttng-ust0 libcurl3 libuuid1 libkrb5-3 zlib1g gnupg2
```
Comment (brendandburns) [inline openapi/openapi-generator/Dockerfile:24]: why is `gnupg` needed?

**Comment by brendandburns** [comment]
One quick question about why the Dockerfile needed to change.

File: openapi/openapi-generator/Dockerfile (hunk 1 above)
Comment (yue9944882) [reply openapi/openapi-generator/Dockerfile]: that's an oversight. thank you for reminding!

**Pull request merged**